kf_lib_data_ingest/common/pandas_utils.py

```diff
@@ -327,6 +327,22 @@ def resolve_duplicates(df, suffixes):
                 if coll.endswith(l_suffix):
                     firstpart = coll.split(l_suffix)[0]
                     colr = firstpart + r_suffix
+                    inconsistent = (
```

Comment:
This logic doesn't quite work. If you try this:
```
df1 = pandas.DataFrame({
    'A': [1, 2, 2],
    'B': [3, 4, 5]
})
df2 = pandas.DataFrame({
    'A': [1, 2, 2],
    'B': [3, None, 6]
})
m = pandas.merge(df1, df2, on='A', how='outer')
```
it passes without exception even though a regular pandas merge shows inconsistent data:

A | B_x | B_y
-- | -- | -- 
1 | 3 | 3.0
2 | 4 | NaN
2 | 4 | 6.0
2 | 5 | NaN
2 | 5 | 6.0

I think you need something like this:
```python
df_no_nulls = df[df[coll].notnull() & df[colr].notnull()] # we only care about rows where both values are not null
inconsistent = df_no_nulls[coll] != df_no_nulls[colr])
```

Reply:
You used pandas.merge instead of merge_wo_duplicates. Try again with the actual function.

Reply:
oh sorry, i thought i hit Start a Review but I had hit Add single comment. I went back and removed my previous comment bc i realized that i wasn't running your branch. I see that your code does work